app/testOnly/controllers/FeatureSwitchController.scala

```diff
@@ -106,4 +106,14 @@ class FeatureSwitchController @Inject()(featureSwitchView: FeatureSwitchView,
 
   }
 
+  def enableAll(): Action[AnyContent] = Action.async { implicit user =>
```

Comment:
What environment do these work on? It would be good to preserve the original state of the FSs if in Staging/QA so that they don't just constantly get turned on?

Reply:
testOnly endpoint available only in staging (and QA?) as far as I know

Reply:
Why not local? What environment do PTs run on? This would be most important I think

Reply:
In local we have "FS read from mongo" feature disabled and this for good; as I understand scope of this story its just to apply this where we have "FS read from mongo" set to on